phpunit.xml.dist

```diff
@@ -1,10 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
-    colors="true"
-    bootstrap="tests/bootstrap.php"
-    backupGlobals="true"
-    convertDeprecationsToExceptions="true"
-    >
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.0/phpunit.xsd"
+         colors="true"
+         bootstrap="tests/bootstrap.php"
+         backupGlobals="true"
+         cacheDirectory=".phpunit.cache">
```

Comment:
@LordSimal Any particular reason for specifying `cacheDirectory` or was it auto added by phpunit's config migrator?